test/integration/emitter_test.cpp

```diff
@@ -1047,6 +1047,37 @@ TEST_F(EmitterTest, BoolFormatting) {
       "- Y\n- Y\n- y\n- N\n- N\n- n");
 }
 
+TEST_F(EmitterTest, GlobalNullFormatting) {
+  out << BeginSeq;
+  out.SetNullFormat(LowerNull);
+  out << Null;
+  out.SetNullFormat(UpperNull);
+  out << Null;
+  out.SetNullFormat(CamelNull);
+  out << Null;
+  out.SetNullFormat(TildeNull);
+  out << Null;
+  out << EndSeq;
+  ExpectEmit("- null\n- NULL\n- Null\n- ~");
```

Comment:
I know the related tests don't do this, but I think it'd be easier to read if this sequence is written in flow style (and below).